src/pages/DashboardPage.tsx

```diff
@@ -69,17 +130,66 @@ const GET_DASHBOARD_NUMBERS = gql`
   }
 `
 
+export interface ClaimsFiltersTypeWithName extends ClaimsFiltersType {
+  name?: string
+}
+
+export interface TemplateFiltersType {
+  filters: ClaimsFiltersTypeWithName[] | []
+}
```

Comment:
This one doesn't really make sense, why not just

```suggestion
export interface TemplateFilters {
  filters: ClaimsFiltersTypeWithName[]
}
```